introspect/README.md

```
@@ -0,0 +1,61 @@
+
+```bash
+cargo build --release
+```
+
+## Usage
+
+```bash
+./target/release/iii-introspect --url ws://127.0.0.1:49134 --config ./config.yaml
+```
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)